inventory_reporting/reports/inventory_report.py

```diff
@@ -106,59 +106,52 @@ def generate_storable_report(self, workbook, wizard):
                 "filter": [
                     ("stock_move_id.picking_type_id.code", "=", "incoming"),
                     ("stock_move_id.origin_returned_move_id", "=", False),
+                    ("stock_move_id.unbuild_id", "=", False),
                 ],
             },
             {
                 "name": _("Return"),
                 "filter": [
+                    "|",
+                    ("stock_move_id.picking_type_id.code", "=", "internal"),
+                    ("stock_move_id.picking_type_id.code", "=", "outgoing"),
```

Comment:
```suggestion
                    ("stock_move_id.picking_code", "in", ("internal", "outgoing")),
```
Do we actually need `internal`?

Reply:
Yes, I think for return mainly they are using Internal transfer operation. but if i check on their DB, sometimes they also directly process the transfer from the related reception.